Formula/j/julia.rb

```diff
@@ -2,11 +2,20 @@ class Julia < Formula
   desc "Fast, Dynamic Programming Language"
   homepage "https://julialang.org/"
   # Use the `-full` tarball to avoid having to download during the build.
-  url "https://github.com/JuliaLang/julia/releases/download/v1.10.5/julia-1.10.5-full.tar.gz"
-  sha256 "5fc94159b87f34d18bf44a1a940e59f40cbeab931bfdc0c4763203bbb6995260"
+  # TODO: Check if we can unbundle `curl`: https://github.com/JuliaLang/Downloads.jl/issues/260
   license all_of: ["MIT", "BSD-3-Clause", "Apache-2.0", "BSL-1.0"]
   head "https://github.com/JuliaLang/julia.git", branch: "master"
 
+  stable do
+    url "https://github.com/JuliaLang/julia/releases/download/v1.11.0-rc4/julia-1.11.0-rc4-full.tar.gz"
+    sha256 "ca39244c1030b177a2ebe013bda5f5871df9223d97e30ddf96e4e24387b99c59"
+
+    patch do
+      url "https://github.com/JuliaLang/julia/commit/0dbb6eb679c1c124c212ae9ce399004873041cf1.patch?full_index=1"
+      sha256 "d2b91c39daaf0181f4088b347f5a8b434dadf56630465819f957d33731f482dc"
+    end
```

Comment:
I think patching triggers a download request which then needs libcurl and ultimately fails. Will drop patch to see if macOS succeeds.

Reply:
> A build flag, `WITH_TERMINFO`, is added here and assumed true by default, allowing users to set `WITH_TERMINFO=0` in `Make.user` to avoid bundling `terminfo` should they want to do so.

Maybe we should just disable this?

Reply:
Can try. `ncurses` is already in dependency tree (thus conflict).